python/lsst/sims/catUtils/utils/ObservationMetaDataGenerator.py

```diff
@@ -131,8 +125,137 @@ def __init__(self, driver=None, host=None, port=None, database=None):
 
         self.dtype = numpy.dtype(dtypeList)
 
+    @staticmethod
+    def OpSimColumnMap(seeing_column):
+        """
+        Return a list of tuples.  Each tuple has the following elements :
+        (phosimName, OpSimName, unknown, dtype, callable)
 
-    def getObservationMetaData(self, obsHistID=None, expDate=None, fieldRA=None, fieldDec=None,
+        Needs some work:
+        """
+        v = [('obsHistID', 'obsHistID', 'Opsim_obshistid', numpy.int64, None),
+             ('expDate', 'expDate', 'SIM_SEED', int, None),
+             ('fieldRA', 'fieldRA', 'pointingRA', float, numpy.radians),
+             ('fieldDec', 'fieldDec', 'pointingDec', float, numpy.radians),
+             ('moonRA', 'moonRA', 'Opsim_moonra', float, numpy.radians),
+             ('moonDec', 'moonDec', 'Opsim_moondec', float, numpy.radians),
+             ('rotSkyPos', 'rotSkyPos', 'Opsim_rotskypos', float, numpy.radians),
+             ('telescopeFilter', 'filter', 'Opsim_filter', (str, 1), lambda x: '\'{}\''.format(x)),
```

Comment:
You appear to have done away with the need for ```_put_quotations()```.  Go ahead and delete that method.